scripts/murmur.ini

```diff
@@ -206,6 +206,11 @@ allowping=true
 ;
 ;rememberchannel=true
 
+; How many seconds should the server remember the last channel of a user.
+; Set to 0 (default) to remember forever. This option has no effect if
+; rememberchannel is set to false.
+;rememberduration=0
```

Comment:
I think calling this `rememberchannelduration` instead makes the intention more clear :thinking: